electron/generators/diory-generator.js

```diff
@@ -1,6 +1,6 @@
 const { v4: uuid } = require('uuid')
 const { resolveFileType, readFile } = require('../readers/file-reader')
-const { readFolder } = require('../readers/folder-reader')
+const { readFolderMetadata } = require('../readers/folder-reader')
 const { readImage } = require('../readers/image-reader')
 const { readVideo } = require('../readers/video-reader')
```

Comment:
Should each reader type generate its complete diory? Could the name be then e.g. image-diory-generator. However, I haven't even understood this "reader, handler, ..." naming convention, the best name for the file and the function would be getImageDiory!

Reply:
It does not matter if it's a reader or generator or whatever.